README.md

```diff
@@ -368,7 +368,7 @@ Parameters of the driver script, `abyss-pe`
  * `q`: minimum base quality [`3`]
  * `s`: minimum unitig size required for building contigs (bp) [`200`]
  * `S`: minimum contig size required for building scaffolds (bp) [`s`]
- * `t`: minimum tip size (bp) [`2k`]
+ * `t`: maximum length of dangling edges to trim [`k`]
```

Comment:
Sounds good.

Unfortunately it looks as though `trimLen` is measured in *k*-mers and `bubbleLen` is measured in bp. Oi. That's not great.